docs/source/guide/index.rst

```diff
@@ -155,56 +156,50 @@ Step 1: Configure the Roles
     https://example.com/downloads/.                                                                     
 
     Additionally, it has two sub-folders, productA and productB where the clients can find all files
-    (i.e.: productA-v1.0.tar, productB-1.0.tar), for productB it even has a sub-folder, updates where
+    (i.e.: productA-v1.0.tar, productB-v1.0.tar), for productB it even has a sub-folder, updates where
     clients can find update files (i.e.: servicepack-1.tar, servicepack-2.tar).
+    The organization has decided to use 8 hash bins. Target files will be uniformly distributed over
+    8 bins whose names will be "1.bins-0.json", "1.bins-1.json", ... , "1.bins-7.json".
 
-    In that case mapping all targets files paths as:                                                    
+    Now imagine that the organization stores the following files:
+    - https://example.com/downloads/productA/productA-v1.0.tar
+    - https://example.com/downloads/productB/productB-v1.0.tar
+    - https://example.com/downloads/productB/updates/servicepack-1.tar
 
-    • https://example.com/downloads/ is *                                                              
-    • https://example.com/downloads/productA/ is */*                                                   
-    • https://example.com/downloads/productB/ is */* (same as above)                                   
-    • https://example.com/downloads/productB/updates/ is */*/*                                         
+    As we said the targets will be uniformly distributed over the 8 bins no matter if they are
+    located in the same folder. In this example here is how they will be distributed:
+    - "1.bins-0.json" will be responsible for file https://example.com/downloads/productA/productA-v1.0.tar
+    - "1.bins-1.json" will be responsible for file https://example.com/downloads/productB/productB-v1.0.tar
+    - "1.bins-5.json" will be responsible for file https://example.com/downloads/productB/updates/servicepack-1.tar
```

Comment:
I would remove the base URL and put only the path, to avoid the user understand that the `path` is full url
```suggestion
    - "1.bins-0.json" will be responsible for file productA/productA-v1.0.tar
    - "1.bins-1.json" will be responsible for file productB/productB-v1.0.tar
    - "1.bins-5.json" will be responsible for file productB/updates/servicepack-1.tar
```

Reply:
Okay, will do and will recalculate where those target roles will be.